docs/source/conf.py

```diff
@@ -0,0 +1,30 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Ara'
+copyright = '2022, Sharafat Hussain'
```

Comment:
We think there can be a problem with specifying a `copyright` here without any licensing information. 
Do you really want to copyright the documentation? If not, you can remove this line and we can merge. Of course, you would remain the author, and you can be acknowledged for it with the `author` line!

Reply:
I agree. I have removed it.